templates/api/knowledge-panels/environment/ecoscore/origins_of_ingredients.tt.json

```diff
@@ -25,11 +25,11 @@
             }
         }, 
 [% ELSE %]
-    [% IF product.ecoscore_data.adjustments.origins_of_ingredients.value <= 0 %]
+    [% IF defined product.ecoscore_data.adjustments.origins_of_ingredients.value && product.ecoscore_data.adjustments.origins_of_ingredients.value <= 0 %]
```

Comment:
There's an error in the template:

(  DIAG  )  job 63    failed to fetch template: file error - parse error - api/knowledge-panels/environment/ecoscore/origins_of_ingredients.tt.json line 28: unexpected token (product)

Reply:
You can't do [% IF defined something && something <= 0 %] in a template, but instead [% IF something.defined && something <= 0 %]